src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -241,16 +460,17 @@ public String processUpload(UploadedFile file, HttpServletRequest request) {
      * @param uniqueId The UploadedFile form backing object containing the file.     *
      * @return The number of blank lines in the file.
      */
+    /*
     @RequestMapping(value = "/getBlankLines", method = RequestMethod.GET)
     @ResponseBody
     public String getBlankLines(@RequestParam("fileName") String fileName, @RequestParam("uniqueId") String uniqueId) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)